examples/aws-atlas-privatelink-regionalized/Readme.md

```diff
@@ -0,0 +1,112 @@
+# Example - AWS and Atlas PrivateLink Regionalized Mode with Terraform
+
+This project aims to provide a very straight-forward example of setting up PrivateLink connection between AWS and MongoDB Atlas. Using `mongodbatlas_private_endpoint_regional_mode` to determine if it's enabled or not.
+
+## Gotchas
+- Ensure `mongodbatlas_cluster` depends_on `mongodbatlas_private_endpoint_regional_mode`
+- Despite properly output, connection strings _are not applied_ properly to `mongodbatlas_cluster` when changing regional mode enabled.
```

Comment:
@evertsd I know we talked about this a bit but can you help me understand what you are trying to get across here?   Are you saying the correct connection strings are output properly but they are not saved in the mongodbatlas_cluster state?  I gave it a try on suggestion so tell me if this is off the mark.
```suggestion
- Despite being properly output, connection strings _are not applied_ properly to `mongodbatlas_cluster` when changing regional mode enabled.   This means they will not exist in the cluster state until the next refresh.  
```

Reply:
> Are you saying the correct connection strings are output properly but they are not saved in the mongodbatlas_cluster state?
Correct

I've updated the language to be the following. In running the example it seems that the connection_strings are often applied correctly on the first `apply`, but in the automated test two applies were necessary.
```
Despite being properly output, connection strings _may not be applied_ to `mongodbatlas_cluster` resource when changing regional mode enabled. This means the connection_strings may not exist in terraform state until the next `terraform apply`.
```

Reply:
That makes sense, thank you!